pkgs/data/misc/media-types/default.nix

```diff
@@ -0,0 +1,65 @@
+{ lib, stdenv, fetchFromGitLab }:
+
+stdenv.mkDerivation rec {
+  pname = "media-types";
+  version = "8.0.0";
+
+  src = fetchFromGitLab {
+    domain = "salsa.debian.org";
+    owner = "debian";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-dl6XKxOF9YT9n/iSvZj1juTqSMaZuOF5i7Eb/VZZDFY";
+  };
+
+  installPhase = ''
+    install -D -m0444 mime.types $out/etc/mime.types
+    install -D -m0444 debian/bug-presubj $out/share/bug/media-types/presubj
+    install -D -m0444 debian/changelog $out/share/doc/media-types/changelog
+    install -D -m0444 debian/copyright $out/share/doc/media-types/copyright
+
+    # Add missing mime-types
+    sed -i $out/etc/mime.types \
+      -e '/^application\/x-wais-source\t\t\tsrc$/aapplication\/x-web-app-manifest+json\t\twebapp' \
+      -e '/^text\/vnd.net2phone.commcenter.command\t\tccc$/atext\/vnd.rim.location.xloc\t\t\txloc' \
+      -e 's/^text\/xml$/text\/xml\t\t\t\t\txsd rng/'
+
+    # Generate nginx mime-types
+    mkdir -p  $out/etc/nginx
+    echo "types {" > $out/etc/nginx/nginx-mime.types
+    cat $out/etc/mime.types | sed -e '/^#.*/d' -e '/^$/d' -e 's/.*/\ \ &;/' -ne '/[^[:space:]][[:space:]]\{1,\}[^[:space:]]/p' >> $out/etc/nginx/nginx-mime.types
+    echo "}" >> $out/etc/nginx/nginx-mime.types
```

Comment:
Can we simplify this?

Reply:
I have no other options. Mailcap uses a similar script:
```
#!/bin/sh
echo "types {"
sed -e 's/#.*//' -e 's/$/;/' -ne '/[^[:space:]][[:space:]]\{1,\}[^[:space:]]/p'
echo "}"
```